src/bootstrap/check.rs

```diff
@@ -570,6 +604,7 @@ static HOST_COMPILETESTS: &[Test] = &[
     },
     Test { path: "src/test/run-make", mode: "run-make", suite: "run-make" },
     Test { path: "src/test/rustdoc", mode: "rustdoc", suite: "rustdoc" },
+    Test { path: "src/test/rustdoc-js", mode: "rustdoc-js", suite: "rustdoc-js" },
```

Comment:
This (probably) shouldn't be here, unless you want to branch on rustdoc-js as the mode below and call into `RustdocJs`... I would remove this, and add `RustdocJs` to the list of tests here https://github.com/rust-lang/rust/blob/ee220daca345302c3277befee2732b6b2a5a711c/src/bootstrap/builder.rs#L254-L257.